docs/source/intro.rst

```diff
@@ -7,19 +7,34 @@ Taweret for Bayesian Model Mixing
 Bayesian Model Mixing
 ---------------------
 
-In computer simulation studies we often come across multiple theoretically \
-sound models that are equally good. These models have different assumptions built into them \
-and are only valid in certain regions of the input domain. It is most certainly possible that the true \
-model that can describe the phenomena is not any of the models that are being considered. Bayesian Model \
-Mixing is a data-driven technique that allows us to combine the models to get the most accurate predictions\
-and calibrations while taking into account the modeling uncertainties. Taweret is a python package \
-for Bayesian Model Mixing. 
-
-Typical workflow of Bayesian Model Mixing
-
-1. Collect all the models and experimental data 
-2. Decide on a mixing method
-3. Find the weights/mixing function to combine models
+In computer simulation studies, it is often the case where a set of theoretically \
+sound models are considered, each intended to describe the true underlying physical phenomena of interest \
+across a sub-region of the domain. These models could differ by their underlying assumptions \
+and may not be valid in certain sub-regions of the input domain. In practice, the true underlying \
+model is not contained in the set of candidate models. Thus, selecting a single model to to describe the true phenomena \
+across the entire input domain is inappropriate. As an alternative, one may elect to combine the information within \
+the model set in some systematic manner. A common approach involves combining the individual \
+mean predictions or predictive densities from the indivdual models using a linear combination or weight4ed average. \
+The weights in this linear combination may or may not depend on the inputs. When the models under consideration \
+exhibit varrying levels of predictive accuracy depending on the sub-region of the input domain, an input-dependent \
+weighting scheme is more appropriate. A memeber of the class of input-dependent weighting schemes is \
+Bayesian Model Mixing (BMM). BMM is a data-driven technique combines the predictions from a set of N candidate models in a \
+Bayesian manner using input-dependent weights. Mixing can be performed using one of two strategies described below: \
+(1) A two-step approach: Each model is fit using one source of data prior to mixing. \
+Conditional on the predictions from each model, the weight functions are learned using a second source of data. \
+(2) A joint analysis: When the models have unknown parameters, one could elect to perform calibration while simultaneously \
```

Comment:
Why do weight have to be learned from a second source? For example in the Coleman thesis BMM example he used the same data for model calibration and then the mixing of models.